src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml

```diff
@@ -48,13 +47,13 @@
               Grid.Row="1"
               Grid.Column="0">
             <Grid Grid.Row="0" Margin="4,7,4,4">
-                <graphControl:Grapher Name="GraphingControl" 
-                                  EquationsSource="{x:Bind ViewModel.Equations, Mode=OneWay}"
-                                  ForceProportionalAxes="True"
-                                  UseSystemFocusVisuals="True"
-                                  VariablesUpdated="GraphVariablesUpdated"
-                                  LostFocus="OnGraphLostFocus"
-                                  LosingFocus="OnLoosingFocus">
+                <graphControl:Grapher Name="GraphingControl"
+                                      EquationsSource="{x:Bind ViewModel.Equations, Mode=OneWay}"
+                                      ForceProportionalAxes="True"
+                                      LosingFocus="OnLosingFocus"
```

Comment:
I wouldn't do that, your making a hungarianish prefixing for the function names which is fragile.  They are in the controls space and so should be pretty clear as to the context.

Reply:
This actually isn't in the graph control space, OnLosingFocus could be referring to any of the controls on this page so it could be useful to give it a more descriptive name.